release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1443,7 +1440,20 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
 
 *Web console (Developer perspective)*
 
+* This update adds an error message to the *Workloads* menu of the monitoring dashboard if a user does not have access privileges. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930546[*BZ#1930546*])
+
+* This update fixes the sort order and color of `ImageManifestVuln` objects for the Quay Security Operator. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942716[*BZ#1942716*])
 
+* Previously, the *Topology* view in the Developer perspective did not load if OpenShift namespace templates were not available. This update fixes the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949810[*BZ#1949810*])
```

Comment:
The BZ gives a [specific reason why there are no templates](https://bugzilla.redhat.com/show_bug.cgi?id=1949810#c2). Is that worth adding here to make the situation less mysterious?